src/models/db/account.js

```diff
@@ -83,7 +83,9 @@ class Account extends Model {
     // some databases store booleans as 0 and 1, and knex does not convert
     data.is_disabled = Boolean(data.is_disabled)
     data.is_admin = Boolean(data.is_admin)
-    data.balance = Number(data.balance)
+    // oracle has balance stored properly, but knex returns it with a small
+    // rounding error, possibly a bug in knex; using toFixed as workaround
+    data.balance = Number(data.balance.toFixed(2))
```

Comment:
toFixed(2) means we will always round to 0.01 of the currency (i.e. to the cent if the currency is USD).  Is this acceptable @yana-novikova ?  Also, if we round balance to toFixed(2) here, is there an implication on other components?  For example, if Connector says "please add this account $1.005" to Ledger, and Ledger adds $1.01, would there be a problem for Connector?